.circleci/config.yml

```diff
@@ -370,7 +379,7 @@ workflows:
           python_version: "3.6"
           pandas_version: "<1"
           requires:
-            - redshift-itest
+            - redshift-s3-itest
```

Comment:
The existing tests in CircleCI use an S3 scratch bucket, so let's label them as such.